articles/active-learning/index.md

```diff
@@ -0,0 +1,150 @@
+![hero](/engineering-education/active-learning/hero.jpg)
+
+[Image Source](https://images.unsplash.com/photo-1546410531-bb4caa6b424d?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1051&q=80)
+
+By nature, machine learning models are data-hungry. They require a lot of data to make decisions. However, always having high quality labeled data to train models is often a costly task. An answer to this would be to use an approach that can reduce the training data needed to train a model. Active learning is such an approach.
+
+### Contents
+
+1. Introduction to active learning
+
+2. Steps to use active learning on unlabelled data
+
+3. Three active learning frameworks
+
+4. Query strategies
+
+### Prerequisites
+
+This article requires an understanding of supervised learning, in addition to machine learning in general. This [article](https://www.section.io/engineering-education/supervised-learning-algorithms/) may be of help with the mentioned prerequisites.
+
+### Active Learning
+
+In supervised learning, a great amount of labeled data is needed for training. However, we might encounter a situation where we have a large pool of unlabelled data. To apply a supervised algorithm on such data, we would need to label every instance. However, this would be a tedious, time-consuming, and expensive process.
+
+However, a semi-supervised approach would allow us to label a few data examples to train a model and still achieve great precision. This is active learning. Active learning describes an approach where data that needs to be labeled to train a model is prioritized. Specifically, the data that is prioritized has the highest impact on the performance of a model. Prioritizing informative data allows one to train a model on such data instead of training it on the whole dataset. We describe this approach as semi-supervised since unsupervised learning approaches consider unlabelled data while supervised ones prefer a fully labeled dataset. Active learning does not need a fully labeled set. Active learning involves the use of an oracle. In this context, an oracle is the information source. That is an entity that is capable of labeling data examples when the learner cannot. This is often a human expert.
```

Comment:
Break this para into smaller paragraphs. 

 We describe this approach as semi-supervised since unsupervised learning approaches consider unlabelled data while supervised ones prefer a fully labeled dataset. -> We describe this approach as semi-supervised. Unsupervised learning approaches consider unlabelled data while supervised ones prefer a fully labeled dataset.